src/pages/edge/adobe-journey-optimizer/public-classes/index.md

```diff
@@ -25,9 +25,14 @@ This documents lists details about the public classes and enums available in Mes
 * [Class - Message](./message.md)
 * [Enum - MessagingEdgeEventType](./messaging-edge-event-type.md)
 
-## Code-based experiences
+## Proposition-based messaging<br /> \(Code-based experiences & Content Cards)
```

Comment:
Why do we have the `<br>` tag here?

Reply:
stylistic reasons.  all the other headings on this page are short, stacking this one made it look less awkward to me :)